src/tests/ftest/util/daos_core_base.py

```diff
@@ -83,7 +83,9 @@ def start_server_managers(self, force=False):
                     env_dict = {
                         item.split("=")[0]: item.split("=")[1]
                         for item in env_vars}
-                    env_dict["CRT_CTX_SHARE_ADDR"] = "1"
+
+                    self.log.error("Scalable endpoint mode not implemented");
```

Comment:
pylint and flake
```suggestion
                    self.log.error("Scalable endpoint mode not implemented")
```

Reply:
Also, why always log an error? Maybe use `log.info` instead?

Reply:
Error log to get noticed. SEP shouldnt be enabled anywhere; this might help catch situations when same test runs twice once with and once without SEP.

Reply:
Should we just remove that functionality then? It looks like only this test uses it:
https://github.com/daos-stack/daos/blob/7c12c85e72ad530db5f152eb648eb454628eb001/src/tests/ftest/daos_test/suite.yaml#L179-L180
https://github.com/daos-stack/daos/blob/7c12c85e72ad530db5f152eb648eb454628eb001/src/tests/ftest/daos_test/suite.single_host.yaml#L145-L146

Reply:
I believe this test was moved to scalable endpoint as it used to be having slowness/perf issues under sockets when running in non-sep mode. So we probably still want 'test_daos_degraded_mode' to run, just not under scalable_endpoint label

Reply:
I don't think the test needs to be moved to a different section - just remove those two config lines and the test will run the same way, but without scalable endpoint